working/2364 - primary constructors/feature-specification.md

```diff
@@ -675,7 +790,8 @@ of the fact that the primary constructor is actually placed outside the
 braces that delimit the class body.*
 
 Next, _k_ has the modifier `const` iff the keyword `const` occurs just
-before the name of _D_ or before `this`, or _D_ is an `enum` declaration.
+before the name of _D_ or before `this`, or if _D_ is an `enum`
+declaration.
```

Comment:
So we infer `const` for header constructors on enums, but not body constructors? That seems like an irregularity.